packages/integrations/sitemap/test/fixtures/ssr/astro.config.mjs

```diff
@@ -3,7 +3,15 @@ import sitemap from '@astrojs/sitemap';
 import { defineConfig } from 'astro/config';
 
 export default defineConfig({
-  integrations: [sitemap()],
+  integrations: [sitemap({
+		customPages: async (site) => {
+			return [
+				`${site}/solutions/`,
+				`${site}/solutions/one/`,
+				`${site}/solutions/two/`,
+			];
+		}
+	})],
```

Comment:
If possible, I would avoid modify the existing tests, because it could hide possible regressions. Instead, create a new fixture so we can test the new feature